distributionviewer/core/static/js/app/components/views/example-chart.js

```diff
@@ -25,30 +26,35 @@ export default function(props) {
       { x: 90,  y: 86 },
       { x: 95,  y: 86 },
       { x: 100, y: 86 },
-    ]
-  }];
+    ];
 
-  return  (
-    <LineChart
-      data={exampleData}
+    MG.data_graphic({
+      target: this.target,
 
-      // General display
-      title="Number of available TV channels"
-      gridHorizontal={false}
-      circleRadius={0}
-      width={350}
-      height={250}
+      // Data
+      data: exampleData,
+      x_accessor: 'x',
+      y_accessor: 'y',
 
-      // Axes
-      domain={{x: [,100], y: [0,100]}}
+      // General display
+      title: "Number of available TV channels",
+      width: 350,
+      height: 250,
+      area: false,
 
       // x-axis
-      xAxisLabel="channels"
-      xAxisTickValues={[0, 5, 100]}
+      x_label: "channels",
```

Comment:
Good catch. I agree we should keep things consistent. In the near future, I'll try to an eslint Gulp task so that we catch these things earlier. In the meantime, I don't mind fixing them before we merge.